### ISSUE TITLE
Question: delete/flag users that no longer match search filter

### ISSUE THREAD

**Issue by frennkie** [etianen/django-python3-ldap#117]
I hope I haven't overlooked this is the documentation or older issues:

I'm using the `LDAP_AUTH_FORMAT_SEARCH_FILTERS` to limit access to LDAP users that are `memberOf` a LDAP group called `web_app_users`. So let's assume `alice`, `bob` and `eve` are member of the `web_app_users` group and I run `manage.py ldap_sync_users` then all three a synced to the Django database.

So far so good.

Now I remove `eve` from the `web_app_users` group and from now on she won't be able to log in anymore. But running `manage.py ldap_sync_users` again will not remove her from the Django database, right?

If this is so, then - on the long term - this will lead to having several users in the Django database that are no longer in the LDAP directory.

A very valid reason not to delete a user immediately from the Django database after she/he is no longer found in LDAP is that there might be additional fields on the user model (e.g. preferences/settings and data). But I think it would make sense to flag her/him somehow, e.g. set the user to `inactive`  in the Django model.

So basically my question is:
* is my understanding correct and 
* where would I put a hook to change a user to `inactive` when she/he is no longer found when `manage.py ldap_sync_users` is run.

**Comment by busla** [comment]
hey mate,

here is a pull request addressing this issue. It might help in your case.

https://github.com/etianen/django-python3-ldap/pull/80
